sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/query.sql

```diff
@@ -1,7 +1,7 @@
 -- Query for telemetry_derived.clients_first_seen_v2
-{% if is_init() and parallel_run() %}
+{% if is_init() %}
 INSERT INTO
-    `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
+ `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v2`
 {% endif %}
```

Comment:
I still think you could avoid the need for this `INSERT INTO` kludge by passing `--append_table` and `--noreplace` arguments to the `bq query` command ([some](https://github.com/mozilla/bigquery-etl/blob/79fa5487c3e5470fb80d12b4a30109fdd3b6030a/script/glam/generate_and_run_desktop_sql#L55) [examples](https://github.com/mozilla/bigquery-etl/blob/a5bcebc30878f0f433653fc9f668ac69f88d01a8/sql/moz-fx-data-shared-prod/stripe_external/customers_changelog_v1/metadata.yaml#L19-L21)).